LANGUAGE.md

```diff
@@ -148,6 +152,23 @@ http.createServer(function(req, res) {
     .listen(8080);
 ```
 
+### amaze
```

Comment:
I wonder if this section should be moved to under Functions

Reply:
Yeah, that makes sense.